rm.go

```diff
@@ -144,9 +141,18 @@ func removeContainer(ctx context.Context, client *containerd.Client, id string,
 		}
 	default:
 		if !force {
-			return errors.Errorf("cannot remove a %v container %v", status.Status, id)
+			var msg string
+			if status.Status == containerd.Paused || status.Status == containerd.Running {
+				if status.Status == containerd.Paused {
+					msg = "Unpause"
+				} else if status.Status == containerd.Running {
+					msg = "Stop"
+				}
+
+				_, err := fmt.Fprintf(clicontext.App.Writer, "Error response from daemon: You cannot remove a %v container %v. %s the container before attempting removal or force remove\n", status.Status, id, msg)
```

Comment:
Not really "Error response from daemon". I don't think the error message needs to 100% correspond to Docker.

Reply:
I agree it doesn't need to be 100% matching to docker 🙂 
I felt it just makes it easier for someone who is transitioning from docker CLI to `nerdctl`

Either way, let me know what do you want me to change it to? and I will update the message.

Reply:
I think we can just remove ""Error response from daemon: " part.

Reply:
Fixed.